samples/unit-tests/breadcrubs/buttons/demo.js

```diff
@@ -0,0 +1,261 @@
+QUnit.test('Breadcrumbs button', function (assert) {
+    const chart = Highcharts.chart('container', {
+        chart: {
+            type: 'column'
+        },
+        xAxis: {
+            type: 'category'
+        },
+        title: {
+            text: null
+        },
+        series: [{
+            name: "Supply",
+            data: [{
+                name: "Fruits",
+                y: 5,
+                drilldown: "Fruits"
+            },
+            {
+                name: "Vegetables",
+                y: 6,
+                drilldown: "Vegetables"
```

Comment:
In this demo, go back to the top level and click on "Vegetables to drill down. Nothing happens even when drilldown is defined below. 

Demo: http://utils.highcharts.local/samples/#test/unit-tests/breadcrubs/buttons

Reply:
In this case, the `drilldown` property is declared for a series but after that, this property isn't mentioned in the `drilldown` object. 
It might be misleading, I will change that.
```
series: [{
            name: "Supply",
            data: [{
                name: "Fruits",
                y: 5,
                drilldown: "Fruits"
            }, {
                name: "Vegetables",
                y: 6,
                drilldown: "Vegetables"
            }, {
                name: "Meat",
                y: 3
            }]
        }],
        drilldown: {
            animation: false,
            series: [{
                name: "Fruits",
                id: "Fruits",
                data: [{
                    name: "Citrus",
                    y: 2,
                    drilldown: "Citrus"
                }, {
                    name: "Tropical",
                    y: 5,
                    drilldown: "Tropical"
                }, ['Other', 1]
                ]
            }]
        }
```